amgut/db/patches/0029.sql

```diff
@@ -0,0 +1,121 @@
+-- October 12, 2015
```

Comment:
Yes, it's dropping plate information, but that information is garbled and not up to date. This table was never used correctly in the old database.